tests/src/whisk/consul/ConsulKVTests.scala

```diff
@@ -57,15 +58,15 @@ class ConsulKVTests extends FlatSpec with Matchers {
         val kvInner = List(("k1", JsNumber(55)), ("k2", JsString("dog")), ("k3/inner", JsString("monkey")))
         val kv = kvInner.map { case (k, v) => (s"$keyPrefix/$k", v) }
         kv foreach { case (k, v) => consul.put(k, v) }
-        retry {
+        retry ({
             val retrieved: Map[String, JsValue] = consul.getRecurse(keyPrefix)
             assert(retrieved.size == kv.size)
             kv.foreach {
                 case (k, v) =>
                     println(s"$k -> $v")
                     assert(retrieved.get(k) == Some(v)) // not a KV operation - no inner retry
             }
-        }
+        }, 5, waitBeforeRetry = Some(5 second))
```

Comment:
Maybe add retries and "timeout" as fields to the testclass instead of using magic numbers?